test/e2e/apimachinery/webhook.go

```diff
@@ -563,11 +571,8 @@ func updateConfigMap(c clientset.Interface, ns, name string, update updateConfig
 	return cm, pollErr
 }
 
-func cleanWebhookTest(f *framework.Framework) {
-	client := f.ClientSet
+func cleanWebhookTest(client clientset.Interface, namespaceName string) {
 	_ = client.AdmissionregistrationV1alpha1().ValidatingWebhookConfigurations().Delete(webhookConfigName, nil)
-	_ = client.AdmissionregistrationV1alpha1().ValidatingWebhookConfigurations().Delete(crdWebhookConfigName, nil)
```

Comment:
Why is this line removed?

Reply:
Hm, probably a rebase mistake

Reply:
Thanks for review, sorry for this, it is indeed a rebase mistake. :(
I will correct it in another pr.

Reply:
Seem @cheftako have fixed this, Thanks @cheftako :)